sound/soc/intel/boards/sof_sdw.c

```diff
@@ -232,6 +257,18 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.ops = &sof_sdw_rt1308_i2s_ops,
 		.init = sof_sdw_rt1308_init,
 	},
+	{
+		.part_id = 0x1316,
+		.direction = {true, true},
+		.dai_name = "rt1316-aif",
+		.init = sof_sdw_rt1316_init,
+	},
+	{
+		.part_id = 0x714,
+		.direction = {false, true},
+		.dai_name = "rt715-aif2",
+		.init = sof_sdw_rt715_init,
```

Comment:
same here, should probably be a different init function.

Reply:
@bardliao is there a reason why you added a new file for sof_sdw_rt1316 but not for rt711-sdca and also not for rt714?

Reply:
> @bardliao is there a reason why you added a new file for sof_sdw_rt1316 but not for rt711-sdca and also not for rt714?

I thought rt711 and rt715 sdca version are basically the same as non-sdca version when I was implementing this. However, I checked with @shumingfan today and found they can be considered as different codecs. So I am going to add a new file for each codec.